interfaces/dom.idl

```diff
@@ -548,7 +548,7 @@ interface DOMTokenList {
   [CEReactions] void add(DOMString... tokens);
   [CEReactions] void remove(DOMString... tokens);
   [CEReactions] boolean toggle(DOMString token, optional boolean force);
-  [CEReactions] void replace(DOMString token, DOMString newToken);
+  [CEReactions] boolean replace(DOMString token, DOMString newToken);
```

Comment:
@lukebjerring, this is a problem we're going to run in pretty quickly with generated IDL, where people update the tests together with the spec. As a reviewer, I think we should probably just say "yes please" and if there are any differences the auto-update will take care of it. WDYT?

Reply:
Presumably, differences would be subtle (whitespace, indenting, etc). Until the updater is mature, and we document how to run it, will just have to accept changes :)